src/components/markdown.js

```diff
@@ -25,8 +25,6 @@ const renderAndSanitizeMarkdown = (text, renderers = {}) => DOMPurify.sanitize(m
 }))
 
 /**
- * WARNING: Be very careful when using custom renderers because they may override marked's built-in
- * content sanitization.
```

Comment:
https://marked.js.org/#usage prominently says:
> Warning: 🚨 Marked does not [sanitize](https://marked.js.org/using_advanced#options) the output HTML. If you are processing potentially unsafe strings, it's important to filter for possible XSS attacks. Some filtering options include [DOMPurify](https://github.com/cure53/DOMPurify) (recommended), [js-xss](https://github.com/leizongmin/js-xss), [sanitize-html](https://github.com/apostrophecms/sanitize-html) and [insane](https://github.com/bevacqua/insane) on the output HTML! 🚨

We use DOMPurify to sanitize the HTML in `renderAndSanitizeMarkdown`.

Reply:
Nicely done.